src/scripts/main.js

```
@@ -1,3 +1,86 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('.field');
+  const appendRowBtn = document.querySelector('.append-row');
+  const removeRowBtn = document.querySelector('.remove-row');
+  const appendColBtn = document.querySelector('.append-column');
+  const removeColBtn = document.querySelector('.remove-column');
```

You query the table and buttons here. If any of these selectors returns null, later code (updateButtons or addEventListener) will throw. Consider verifying these variables are non-null before proceeding and failing early or returning with a clear message. This ensures safe usage of `.disabled` and `.addEventListener` (checklist items 2.2, 2.3).